tests/unit/test_vendor_tornado.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+# Import Python libs
+from __future__ import absolute_import, unicode_literals
+import os
+import re
+import logging
+
+
+# Import Salt Testing libs
+from tests.support.unit import TestCase, skipIf
+from tests.support.runtests import RUNTIME_VARS
+
+# Import Salt libs
+import salt.modules.cmdmod
+import salt.utils.platform
+
+log = logging.getLogger(__name__)
+
+
+@skipIf(not salt.utils.path.which('bash'), 'Bash needed for this test')
+class VendorTornadoTest(TestCase):
+    '''
+    Ensure we are no using any non vendor'ed tornado
```

Comment:
Simple typo here:
`Ensure we are not using any non vendor'ed tornado`

Reply:
Fixed